source/stylesheets/about.css.scss

```diff
@@ -86,6 +86,16 @@ body.index, body.about {
   @media screen and (min-width: 2000px) { // Stretch background width when wider than source image
```

Comment:
since I'm seeing inconsistencies across the entire codebase, it will help us to do this & add in documentation for our styles.